kong/dao/schemas/apis.lua

```diff
@@ -16,11 +17,38 @@ local function validate_target_url(value)
   return false, "Invalid target URL"
 end
 
+local function check_public_dns_and_path(value, api_t)
+  local public_dns = type(api_t.public_dns) == "string" and stringy.strip(api_t.public_dns) or ""
+  local path = type(api_t.path) == "string" and stringy.strip(api_t.path) or ""
```

Comment:
What I'm talking about is let's say that I pass the value `/path/]` for example. While that would validate properly here, when attempting to do the match later when trying to find a matching path if that is used as a regular expression it wouldn't compile properly due to an invalid character (the lonely `]`). 

I bring it up because I've run into a similar thing in the past and I just didn't think about it at all. Not trying to say this code is incorrect, just pointing out a potential pitfall.